ostree-ng.sh

```diff
@@ -485,13 +489,129 @@ greenprint "📥 Downloading the image"
 sudo composer-cli compose image "${COMPOSE_ID}" > /dev/null
 ISO_FILENAME="${COMPOSE_ID}-${INSTALLER_FILENAME}"
 modksiso "${ISO_FILENAME}" "/var/lib/libvirt/images/${ISO_FILENAME}"
```

Comment:
Remove this line please.

Reply:
Ok, I will amend the commit. 

One question: Both BIOS and UEFI VMs use 
--cdrom "/var/lib/libvirt/images/${ISO_FILENAME}" (lines 641 and 703)
Should I change them to --cdrom "${ISO_FILENAME}"? 
Or copy ${ISO_FILENAME} to "/var/lib/libvirt/images/${ISO_FILENAME}"?

Reply:
OKey! I see. Keep it. Thanks.